### ISSUE TITLE
Add an example with GNN

### ISSUE THREAD

**Issue by robertanto** [OpenBioLink/OpenBioLink#51]
It may be useful to add an example with a Graph Neural Network (PyTorch Geometric, DGL, etc.).

**Comment by LFGMUW** [comment]
Try using the DGL Framework with the R-GCN modified for big data (mini batch training) 
[rgcn_mb.zip](https://github.com/OpenBioLink/OpenBioLink/files/5580243/rgcn_mb.zip)
on the OpenBioLink data files fitting the load_obldata.py names, with a command like 
python3 link_predmb.py --low-mem --lr 0.00165 --n-bases 2 --n-layers 1 --n-epochs 30 --fanout -1 --batch-size 512 --regularization-coef 3e-6 --valid-neg-cnt 1000 --test-neg-cnt -1 --use-self-loop --num-worker 4 --n-hidden 150 --dropout 0.2 --dataset hc  --global-norm –gpu 4
